trie-db/src/iterator.rs

```diff
@@ -496,6 +636,52 @@ impl<L: TrieLayout> TrieDBRawIterator<L> {
 		None
 	}
 
+	pub fn next_back_item(&mut self, db: &TrieDB<L>) -> Option<TrieItem<TrieHash<L>, CError<L>>> {
```

Comment:
Should be possible to factor code with `next_item` (eg have a internal non public function that take a direction as parameter).

Reply:
I was thinking of leaving `next_back_item` to avoid a breaking change in regards to the `next_item` api, since it's public function.